Unity-Technologies/com.unity.multiplayer.samples.coop
# feat: teleport mode cheat

### Description (*)

This PR adds a cheat to make the player teleport when clicking instead of moving. This cheat can be toggled on or off.
### Related Pull Requests

### Issue Number(s) (*)

Fixes issue(s): [MTT-2211](https://jira.unity3d.com/browse/MTT-2211)
### Manual testing scenarios

1. Start a game and progress to BossRoom scene
2. Open the cheat window and use the "Toggle Teleport Mode" button
3. Your character should now teleport where you click
### Questions or comments

### Contribution checklist
